fdbsyncd/fdbsync.cpp

```diff
@@ -0,0 +1,604 @@
+#include <string>
+#include <netinet/in.h>
+#include <netlink/route/link.h>
+#include <netlink/route/neighbour.h>
+#include <netlink/route/link/vxlan.h>
+#include <arpa/inet.h>
+
+#include "logger.h"
+#include "dbconnector.h"
+#include "producerstatetable.h"
+#include "ipaddress.h"
+#include "netmsg.h"
+#include "macaddress.h"
+#include "exec.h"
+#include "shellcmd.h"
+#include "fdbsync.h"
+#include "warm_restart.h"
+#include "nlapi.h"

```

Comment:
Is this `nlapi.h` is related to any other PRs, thanks

Reply:
I will remove nlapi.h  as that will come in other PR